### ISSUE TITLE
[SPEC] Disparity on facet name: DataQualityMetricsInputDatasetFacet

### ISSUE THREAD

**Issue by howardyoo** [OpenLineage/OpenLineage#1416]
# Problem
According to the spec definition of DataQualityMetricsInputDatasetFacet : https://github.com/OpenLineage/OpenLineage/blob/main/spec/facets/DataQualityMetricsInputDatasetFacet.json, the name of the property is defined as `dataQualityMetrics` here: https://github.com/OpenLineage/OpenLineage/blob/ab26286e57d386b31eb9a12096b318e44596f6e5/spec/facets/DataQualityMetricsInputDatasetFacet.json#L65.

However, in the examples shown here: https://openlineage.io/docs/client/python/#full-example-source-code, the name is defined as `dataQuality`, not `dataQualityMetrics`
```python
# generates input dataset
def inputDataset(dataset, dq):
    input_facets = {
        "dataQuality": dq,
    }
    return InputDataset(dataset.namespace, dataset.name, dataset.facets, input_facets)
```

Either the spec definition is not correct, or example is not correct.
However, when tested with Marquez, looks like `dataQuality` is the correct one.

# Solution
- Clarify whether the name of the above facet is `dataQuality` or `dataQualityMetrics`.
- Fix either the example or spec file accordingly, to remove the confusion/error.

**Comment by mobuchowski** [comment]
I think we corrected it at some point to give Marquez and other consumers chance to move to proper value:

https://github.com/OpenLineage/OpenLineage/blob/ab26286e57d386b31eb9a12096b318e44596f6e5/integration/airflow/openlineage/airflow/extractors/sql_check_extractors.py#L143

but we haven't followed up on that.

**Comment by howardyoo** [comment]
> I think we corrected it at some point to give Marquez and other consumers chance to move to proper value:
> 
> https://github.com/OpenLineage/OpenLineage/blob/ab26286e57d386b31eb9a12096b318e44596f6e5/integration/airflow/openlineage/airflow/extractors/sql_check_extractors.py#L143
> 
> but we haven't followed up on that.

So, is the `dataQualityMetrics` the correct value?

**Comment by mobuchowski** [comment]
@howardyoo yes, given that our naming scheme is `name of facet` + `type of object it's attachable to` + `"Facet"`.

`dataQuality` would be good name for `DataQualityInputDatasetFacet`, not `DataQualityMetricsInputDatasetFacet`.
